File: itcloud/src/main/java/com/itinfo/itcloud/controller/admin/UserController.java
package com.itinfo.itcloud.controller.admin;

import com.itinfo.itcloud.service.admin.ItUserService;
import io.swagger.annotations.Api;
import lombok.RequiredArgsConstructor;
import lombok.extern.slf4j.Slf4j;
import org.springframework.stereotype.Controller;

@Controller
@RequiredArgsConstructor
@Api(tags = "user")
@Slf4j
public class UserController {
	private final ItUserService itUserService;
}

File: itcloud/src/main/java/com/itinfo/itcloud/controller/admin/UserController.kt
package com.itinfo.itcloud.controller.admin

import com.itinfo.common.LoggerDelegate
import com.itinfo.itcloud.service.admin.ItUserService
import io.swagger.annotations.Api
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.RequestMapping

@Controller
@Api(tags = ["user"])
@RequestMapping("/api/v1/")
class UserController {
	@Autowired private lateinit var user: ItUserService
	
	companion object {
		private val log by LoggerDelegate()
	}
}